.github/workflows/test.yml

```diff
@@ -89,4 +89,4 @@ jobs:
               with:
                   name: test_results
                   path: test_results/
-                  if-no-files-found: error
+                  if-no-files-found: warning
```

Comment:
to whom it might concern, I am tired of this error and it's about the time it goes away as it's nothing we changed so we need a story to properly fix this.

Reply:
**amend** that's an `ignore` now

Reply:
apparently this didn't work anyway 🤦‍♂️ ... rolling back, as I keep receiving those notifications I am trained to ignore at this time.